README.md

```diff
@@ -42,12 +44,15 @@ ansible-galaxy collection install -r requirements.yml
 An installation of RH SSO/Keycloak/AWS Cognito must be provided to allow for integration with containerized Trustification.
 
 ## Provision
+
 #### https://developer.hashicorp.com/vagrant/docs/provisioning/ansible
+
 #### https://docs.ansible.com/ansible/2.9/scenario_guides/guide_vagrant.html
 
 In order to deploy Trustification on a RHEL 9.2+ VM:
 
-1. Create an `inventory.ini` file  in the project with a single VM in the `trustification` group:
+1. Create an `inventory.ini` file in the project with a single VM in the `trustification` group:
```

Comment:
Just note: Ansible also supports inventories generated by executable, a.k.a dynamic inventories: https://docs.ansible.com/ansible/latest/inventory_guide/intro_dynamic_inventory.html#intro-dynamic-inventory

Reply:
@i386x, good to know. Not sure if that's going to help much for TPA Single Node deployment though.
I could definitely be handy when doing TPA multiple nodes deployments.